### PR TITLE
Fixed EditorServiceTest

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
Fixed EditorServiceTest

### CODE REVIEW

File: services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java [hunk 1]
```diff
@@ -64,6 +73,9 @@ protected void beforeClass() {
         statusPayload.put("url", "localhost");
         statusPayload.put("error", 0);
         statusPayloadJson = "{ \"key\": \"" + key + "\", \"status\": 1, \"url\": \"localhost\", \"error\": 0 }";
+        RequestLifeCycle.end();
```
Comment (pnedonosko) [inline services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java:76]: Do this in finally block

File: services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java [hunk 2]
```diff
@@ -64,6 +73,9 @@ protected void beforeClass() {
...
+        ExoContainerContext.setCurrentContainer(contextContainer);
+        
       } catch (Exception e) {
         e.printStackTrace();
```
Comment (pnedonosko) [inline services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java:80]: A better reporting (to logger) required - we want see this in surefire reports.

Comment (VladMartyshevskyi) [reply services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java:88]: Added logs: https://github.com/exo-addons/onlyoffice/pull/46/commits/3dc278bea4f2bb0b3b68ba57f5c605dfa5c54fe0

File: services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply services/src/test/java/org/exoplatform/onlyoffice/EditorServiceTest.java]: Added finally block: https://github.com/exo-addons/onlyoffice/pull/46/commits/3dc278bea4f2bb0b3b68ba57f5c605dfa5c54fe0

**Pull request merged**